packages/devtools_app/lib/src/screens/memory/panes/diff/controller/diff_pane_controller.dart

```diff
@@ -36,28 +35,27 @@ class DiffPaneController {
   DiffListItem get selectedItem => snapshots.value[selectedIndex.value];
 
   /// Full name for the selected class.
-  ValueListenable<String?> get selectedClass => _selectedClass;
-  final _selectedClass = ValueNotifier<String?>(null);
-  void setSelectedClass(String? value) => _selectedClass.value = value;
+  final selectedClassName = ValueNotifier<String?>(null);
 
   /// True, if the list contains snapshots, i.e. items beyond the first
   /// informational item.
   bool get hasSnapshots => snapshots.value.length > 1;
 
-  final snapshotStatsSorting = ColumnSorting();
-
-  final classStatsSorting = ColumnSorting();
+  // This value should never be reset. It is incremented for every snapshot that
+  // is taken, and is used to assign a unique id to each [SnapshotListItem].
+  int _snapshotId = 0;
```

Comment:
It is not how it is supposed to be. 
If you take two snapshots, then delete last and then take a snapshot again, you should end up with numbers 1, 2 not 1, 3, as it is more readable.

I think i need to add test for it.